### PR TITLE
Fixes fragile unit and functional tests.

### PR DESCRIPTION
**Pull request by hhamon**

### CODE REVIEW

File: tests/AppBundle/Controller/AdherentControllerTest.php [hunk 1]
```diff
@@ -15,9 +15,12 @@
 use AppBundle\Repository\MailjetEmailRepository;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
-use Tests\AppBundle\SqliteWebTestCase;
+use Tests\AppBundle\MysqlWebTestCase;
 
-class AdherentControllerTest extends SqliteWebTestCase
+/**
+ * @group functional
```
Comment (rpg600) [inline tests/AppBundle/Controller/AdherentControllerTest.php]: so we keep `functionnal` or `functional` ?

Comment (hhamon) [inline tests/AppBundle/Controller/AdherentControllerTest.php]: `functionnal` for. But at some point we'll fix it to `functional`.

File: tests/AppBundle/Controller/EventControllerTest.php [hunk 2]
```diff
@@ -70,32 +79,33 @@ public function testAnonymousUserCanRegisterToEvent()
         $this->assertTrue($this->seeMessageSuccesfullyCreatedFlash($crawler, "Votre inscription à l'événement est confirmée."));
         $this->assertContains('Votre participation est bien enregistrée !', $crawler->filter('.committee-event-registration-confirmation p')->text());
 
-        $crawler = $this->client->request(Request::METHOD_GET, $committeeUrl);
+        $crawler = $this->client->click($crawler->selectLink("Retour à l'événement")->link());
```
Comment (rpg600) [inline tests/AppBundle/Controller/EventControllerTest.php]: single quote

File: tests/AppBundle/Controller/AdherentControllerTest.php (hunk 1 above)
Comment (tgalopin) [inline tests/AppBundle/Controller/AdherentControllerTest.php]: In English, it's `functional`

**Pull request merged**